.gitlab-ci.yml

```diff
@@ -180,7 +180,7 @@ after_script:
   dependencies:
     - build:base
   variables: &ci-template-vars
-    TEST_TARGET: "$CI_JOB_NAME"
+    TEST_TARGET: "${CI_JOB_NAME#*:}"
```

Comment:
This did not work in CI even if I tested and it works well in bash and dash. @SkySkimmer: do you have an idea on what the problem is?

Reply:
I did not understand what you meant but I realized that the `TEST_TARGET` indirection (inherited from Travis) is unnecessary.

Reply:
Oops I misread and thought this was the artifact name key.

Reply:
The point is that gitlab doesn't do any of the `${}` expansions, only the basic `$foo` one (I think it doesn't even have the `${foo}` without modifiers)